src/caustics/lenses/pseudo_jaffe.py

```diff
@@ -294,20 +257,8 @@ def potential(
         """
         x, y = translate_rotate(x, y, x0, y0)
         R_squared = x**2 + y**2 + self.s
-        coeff = (
-            -2
-            * self.get_convergence_0(z_s, params)
-            * core_radius
-            * scale_radius
-            / (scale_radius - core_radius)
-        )
-        return coeff * (
-            (scale_radius**2 + R_squared).sqrt()
-            - (core_radius**2 + R_squared).sqrt()
-            + core_radius * (core_radius + (core_radius**2 + R_squared).sqrt()).log()
-            - scale_radius
-            * (scale_radius + (scale_radius**2 + R_squared).sqrt()).log()
-        )
+        coeff = -2 * self.get_convergence_0(z_s, params) * core_radius * scale_radius / (scale_radius - core_radius)  # fmt: skip
+        return coeff * ((scale_radius**2 + R_squared).sqrt() - (core_radius**2 + R_squared).sqrt() + core_radius * (core_radius + (core_radius**2 + R_squared).sqrt()).log() - scale_radius * (scale_radius + (scale_radius**2 + R_squared).sqrt()).log())  # fmt: skip
```

Comment:
Again this equation is too long, we should simplify it in terms of variables with proper units.

Reply:
I think I've split this one out. Though now it has been revealed that there are units of Mpc for the potential, so I've added a TODO here